pkgs/applications/virtualization/qemu/default.nix

```diff
@@ -233,6 +236,25 @@ stdenv.mkDerivation rec {
   # Builds in ~3h with 2 cores, and ~20m with a big-parallel builder.
   requiredSystemFeatures = [ "big-parallel" ];
 
+  emitKvmWarnings = ''
+    #!${runtimeShell}
+    WARNCOL='\033[1;35m'
+    NEUTRALCOL='\033[0m'
+    WARNING="''${WARNCOL}warning:''${NEUTRALCOL}"
+    if [ ! -e /dev/kvm ]; then
+      echo -e "''${WARNING} KVM is not installed - execution will be slow" >&2
```

Comment:
This should be "loaded" or "available", not "installed", because the presence of KVM can be changed at runtime by inserting or removing the module.

Reply:
I have updated the wording to include both the module not being loaded and KVM not being installed.